Sources/Public/DotLottie/DotLottieFileHelpers.swift

```diff
@@ -0,0 +1,204 @@
+//
+//  DotLottieFileHelpers.swift
+//  Lottie
+//
+//  Created by Evandro Hoffmann on 20/10/22.
+//
+
+import Foundation
+
+extension DotLottieFile {
+
+  // MARK: Public
+
+  /// A closure for an Animation download. The closure is passed `nil` if there was an error.
+  public typealias DotLottieDownloadClosure = (DotLottieFile?) -> Void
+
+  // MARK: DotLottie file (Loading)
+
+  /// Loads a DotLottie model from a bundle by its name. Returns `nil` if a file is not found.
+  ///
+  /// - Parameter name: The name of the lottie file without the lottie extension. EG "StarAnimation"
+  /// - Parameter bundle: The bundle in which the lottie is located. Defaults to `Bundle.main`
+  /// - Parameter subdirectory: A subdirectory in the bundle in which the lottie is located. Optional.
+  /// - Parameter dotLottieCache: A cache for holding loaded lotties. Defaults to `LRUDotLottieCache.sharedCache`. Optional.
+  ///
+  /// - Returns: Deserialized `DotLottie`. Optional.
+  public static func named(
```

Comment:
Can we make this `async` since it's doing some heavy file operations? We don't want consumers to be calling this from the main thread.